assets/js/script.js

```diff
@@ -0,0 +1,353 @@
+var grid =[]
+var total = 0
+var timer = 100
+var gridNo = 6
+var $box = $('.box')
+var $scoreBox = $('.scoreTotal')
+var $gridbox = $('.gridbox')
+var toBeDelete = []
+var toSwitchTwo = []
+var $character = $('.character')
+var $start = $('#start')
+var toBeDeleteFinal =[]
+var $instructionBtn = $('.instructionBtn')
+var $instruction = $('.instruction')
+var iValueOne = 0
+var jValueOne = 0
+var iValueTwo = 0
+var jValueTwo = 0
+var $oneBox =$('#box')
+
+$(function(){
+  var gameTimer = 0
+  var settingMode = true
+  var gameOver=false
+  var gameOverCheck = 0
+  $instruction.hide()
+  createId()
+  $gridbox.css({"height":`${gridNo*95+20}px`,"width":`${gridNo*95+20}px`})
+  $instructionBtn.on('click',function(){
+    $instruction.slideToggle(100,function(){
+    })
+  })
+  $start.on('click',()=>{
+
+    if (settingMode){
+      $('.overText').remove()
+      $('.totalScore').remove()
+      $('.characterFinal').remove()
+      $character.css('background-image','url("./assets/img/yellowteeth.png")')
+      gameOver = false
+      setGrid()
+      gameTimer = setInterval(()=>{
+        timer-= 1
+        $('.timer').text(`${timer}`)
+        settingMode = false
+      }, 1000)
+    }
+    gameOverCheck = setInterval(()=>{
+      if (timer <= 0){
+        gameOver = true
+        clearInterval(gameTimer)
+        endGame()
+        settingMode = true
+        timer = 100
+        clearInterval(gameOverCheck)
+      }
+    },1000)
+  })
+  function createId (){
+    for (var i = 0; i < gridNo; i++) {
+      for (var j = 0; j < gridNo; j++) {
+        $div = $(`<div id="box${i}${j}">`)
+        $gridbox.append($div)
+      }
+    }
+  }
+  function noAvailableMove  (){
+    $gridbox.css({"filter": "grayscale(1)"})
+    $noMove = $('<div class="noMove">')
+    $noMoveTextOne = $('<h2 class="noMoveText">No more moves.</h3>')
```

Comment:
<h2 class> opening tag but <h3> closing tag works? o.o legit question

Reply:
Forgot to change it. Supposed to be h2. But it still appear... why?